source/config/__init__.py

```diff
@@ -65,6 +65,7 @@ def validateConfig(configObj,validator,validationResult=None,keyList=None):
 	#possible log levels are DEBUG, IO, DEBUGWARNING, INFO
 	loggingLevel = string(default="INFO")
 	showWelcomeDialogAtStartup = boolean(default=true)
+	showSpeechViewerAtStartup = boolean(default=false)

```

Comment:
Perhaps move this option into a separate speechViewer section, as we are planning to also save position information as well.